base/main.cpp

```diff
@@ -356,7 +356,7 @@ static void setupGraphics(OSystem &system) {
 		system.setShader(ConfMan.get("shader").c_str());
 		system.setScaler(ConfMan.get("scaler").c_str(), ConfMan.getInt("scale_factor"));
 
-		system.initSize(320, 200);
+		system.initSize(320, 200, Graphics::PixelFormat::createFormatCLUT8());
```

Comment:
This gives the impression the launcher is using CLUT8, but that we forget to set the palette. Actually the format is not used since the overlay has its own pixel format, so maybe it would be clearer to use a null pixel format with `Graphics::PixelFormat()`?